students/FredB/session04/dict_lab.py

```diff
@@ -0,0 +1,49 @@
+#Dict_lab
+#FredBallyns
+#Session04
+
+#Dictionaries 1
+dict_data = {"name": "Fred","city": "Seattle","cake": "Carrot"}
+
+print(dict_data)
+
+dict_data.pop("cake")
+
+print(dict_data)
+
+dict_data["Fruit"] = "Mango"
+
+print(dict_data)
+print(dict_data.keys())
+print(dict_data.values())
+
+print("cake" in dict_data)
+
+print("Fruit" in dict_data)
```

Comment:
Close, we're not necessarily looking to see if "Fruit" exists as a key but that "Mango" exists as a value. In this specific case, this is the same